qiskit/circuit/quantumcircuit.py

```diff
@@ -1533,6 +1534,381 @@ def _rebind_definition(self, instruction, parameter, value):
+        from .library.standard_gates.u3 import U3Gate
+        return self.append(U3Gate(theta, phi, lam), [qubit], [])
+
+    @deprecate_arguments({'ctl': 'control_qubit',
+                          'tgt': 'target_qubit'})
+    def cu3(self, theta, phi, lam, control_qubit, target_qubit, *, label=None, ctrl_state=None,
+            ctl=None, tgt=None):  # pylint: disable=unused-argument
+        """Apply :class:`~qiskit.circuit.library.standard_gates.CU3Gate`."""
+        from .library.standard_gates.u3 import CU3Gate
+        return self.append(CU3Gate(theta, phi, lam, label=label, ctrl_state=ctrl_state),
+                           [control_qubit, target_qubit], [])
+
+    @deprecate_arguments({'q': 'qubit'})
+    def x(self, qubit, *, label=None, ctrl_state=None, q=None):  # pylint: disable=unused-argument
+        """Apply :class:`~qiskit.circuit.library.standard_gates.XGate`."""
+        from .library.standard_gates.x import XGate
+        return self.append(XGate(label=label), [qubit], [])
+
+    @deprecate_arguments({'ctl': 'control_qubit',
+                          'tgt': 'target_qubit'})
+    def cx(self, control_qubit, target_qubit, *, label=None, ctrl_state=None,
+           ctl=None, tgt=None):  # pylint: disable=unused-argument
+        """Apply :class:`~qiskit.circuit.library.standard_gates.CXGate`."""
+        from .library.standard_gates.x import CXGate
+        return self.append(CXGate(label=label, ctrl_state=ctrl_state),
+                           [control_qubit, target_qubit], [])
+
+    @deprecate_arguments({'ctl': 'control_qubit',
+                          'tgt': 'target_qubit'})
+    def cnot(self, control_qubit, target_qubit, *, label=None, ctrl_state=None,
+             ctl=None, tgt=None):  # pylint: disable=unused-argument
+        """Apply :class:`~qiskit.circuit.library.standard_gates.CXGate`."""
+        self.cx(control_qubit, target_qubit, ctl=ctl, tgt=tgt)
+
+    def dcx(self, qubit1, qubit2):
+        """Apply :class:`~qiskit.circuit.gate.DCXGate`."""
+        from .library.standard_gates.dcx import DCXGate
+        return self.append(DCXGate(), [qubit1, qubit2], [])
+
+    @deprecate_arguments({'ctl1': 'control_qubit1',
+                          'ctl2': 'control_qubit2',
+                          'tgt': 'target_qubit'})
+    def ccx(self, control_qubit1, control_qubit2, target_qubit,
+            *, ctl1=None, ctl2=None, tgt=None):  # pylint: disable=unused-argument
+        """Apply :class:`~qiskit.circuit.library.standard_gates.CCXGate`."""
+        from .library.standard_gates.x import CCXGate
+        return self.append(CCXGate(),
+                           [control_qubit1, control_qubit2, target_qubit], [])
+
+    @deprecate_arguments({'ctl1': 'control_qubit1',
+                          'ctl2': 'control_qubit2',
+                          'tgt': 'target_qubit'})
+    def toffoli(self, control_qubit1, control_qubit2, target_qubit,
```

Comment:
should we give a deprecation warning here? I think we should as we should make it clear that these are not the way we want them done.

Reply:
i'll make an issue about aliases in general as i do agree there's a danger of too many methods

Reply:
`toffoli`-`ccx`, `id`-`i` (wire), `cx`-`cnot`, `fredkin`-`cswap`